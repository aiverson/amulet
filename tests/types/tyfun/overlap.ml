type function foo 'a begin
  foo int = string
  foo int = bool
end
