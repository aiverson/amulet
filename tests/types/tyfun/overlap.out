overlap.ml[2:3 ..2:18]: error (E2021)
  Overlapping equations for foo int
  • Note: first defined here,
  │ 
2 │   foo int = string
  │   ^^^^^^^^^^^^^^^^
   but also defined here
  │ 
3 │   foo int = bool
  │   ^^^^^^^^^^^^^^
overlap.ml[2:3 ..2:18]: Use amc explain 2021 for more information
